app/templates/codelists-management/codelist.hbs

```diff
@@ -0,0 +1,110 @@
+{{page-title (t "codelist.details")}}
+
+<BreadcrumbsItem as |linkClass|>
+  <LinkTo
+    @route="codelists-management.codelist"
+    @model={{@model.codelist.id}}
+    class={{linkClass}}
+  >
+    {{@model.codelist.readableLabel}}
+  </LinkTo>
+</BreadcrumbsItem>
+
+<div class="au-o-grid au-o-grid--flush au-o-grid--fixed">
+  <div
+    class="au-o-grid__item {{
+    if
+      this.showSidebar
```

Comment:
I don't think this exists in the controller? If not, I would strip out all this sidebar code. It's confusing to read code that isn't actually used.